### DISCUSSION TITLE
The "Helper Script" is not installed.

### DISCUSSION THREAD

**Discussion by YaGugijuma** [Guilouz/Creality-Helper-Script-Wiki#520] [Issues]
When installing the "Helper Script" it gives an error.

> root@K1-A14C /root [#] git clone https://github.com/Guilouz/Creality-Helper-Scri
> pt.git /usr/data/helper-script
> Cloning into '/usr/data/helper-script'...
> remote: Enumerating objects: 443, done.
> remote: Counting objects: 100% (69/69), done.
> remote: Compressing objects: 100% (50/50), done.
> error: RPC failed; curl 18 transfer closed with outstanding read data remaining
> fatal: the remote end hung up unexpectedly
> fatal: early EOF
> fatal: index-pack failed
> root@K1-A14C /root [#]

**Comment by zlodei2000** [comment]
same error

**Comment by Guilouz** [comment]
It's an network/download issue, not a script issue.
